Sources/Workspace/InitPackage.swift

```diff
@@ -280,8 +292,44 @@ public final class InitPackage {
         }
     }
 
+    private func writePlugins() throws {
+        switch packageType {
+        case .commandPlugin:
+            let plugins = destinationPath.appending(component: "Plugins")
+            guard self.fileSystem.exists(plugins) == false else {
+                return
+            }
+            progressReporter?("Creating \(plugins.relative(to: destinationPath))/")
+            try makeDirectories(plugins)
+
+            let moduleDir = plugins.appending(component: "\(pkgname)")
+            try makeDirectories(moduleDir)
+
+            let sourceFileName = "\(typeName).swift"
+            let sourceFile = try AbsolutePath(validating: sourceFileName, relativeTo: moduleDir)
+
+            let content = """
+                import PackagePlugin
+
+                @main
+                struct \(typeName): CommandPlugin {
+                    func performCommand(context: PluginContext, arguments: [String]) async throws {
+                        print("Hello, World!")
+                    }
+                }
+            """
+
+            try writePackageFile(sourceFile) { stream in
+                stream.write(content)
+            }
+
+        case .empty, .library, .executable, .systemModule, .manifest:
```

Comment:
nit: `default` instead?

Reply:
I remember certain style guides prefer listing out all of the cases explicitly, especially if there's not too many of them. That way, if a new case is added to an enum, you get a compiler error reminding to handle it properly. With `default` such addition of a new case may go unnoticed.

Reply:
Right, that was my rationale here.

Reply:
that is true, expect that in this case do we expect to have any other cases that need to be handled? this seems like command plugin specific. maybe the right way to model this code is to the have a big switch with all the cases that call into into more specific functions eg writeCommandPlugin etc

Reply:
Yes, I expect this function to also be able to generate other types of plugins if we think having these templates is a good idea.

Reply:
gotcha. I would choose a more case-specific code flow, but its not a strong opinion